src/libraries/System.Security.Cryptography.X509Certificates/tests/RevocationTests/DynamicRevocationTests.cs

```diff
@@ -1060,79 +1060,6 @@ public static void CheckIntermediateWithExpiredRevocation(PkiOptions pkiOptions)
                 });
         }
 
-        [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/31249", PlatformSupport.AppleCrypto)]
-        public static void TestRevocationWithNoNextUpdate_NotRevoked()
```

Comment:
I'm a little apprehensive about deleting this test. While your commit message makes sense, we have seen this in the wild: https://github.com/dotnet/runtime/issues/40708.

Reply:
Hm.  I guess I can refactor it to use AsnWriter when OmitNextUpdate is true and use the new type for the 99.9% of other tests.